src/templates/location.tsx

```diff
@@ -58,25 +58,15 @@ export const transformProps = async (data) => {
   const entityConfigurations = document.c_visualConfigurations ?? [];
   const entityLayoutConfigurations = document.c_pages_layouts ?? [];
   const siteLayoutConfigurations = document._site?.c_visualLayouts;
-  try {
-    const templateData = getTemplatePuckData(
-      entityConfigurations,
-      entityLayoutConfigurations,
-      siteLayoutConfigurations,
-      config.name,
-    );
-    const visualTemplate = JSON.parse(templateData);
-    return {
-      ...data,
-      document: {
-        ...document,
-        visualTemplate,
-      },
-    };
-  } catch (error) {
-    console.error("Failed to parse visualTemplate: " + error);
-    return data;
-  }
+  const visualEditorData = resolveVisualEditorData(entityConfigurations, entityLayoutConfigurations, siteLayoutConfigurations, "location");
+  const visualTemplate = JSON.parse(visualEditorData);
```

Comment:
Oh oops do we need this JSON.parse? if so I have to edit the scaffolding template haha

Reply:
Yeah I think so, visualEditorData returns a string